src/Alert/Alert.stories.js

```diff
@@ -1,32 +1,143 @@
 import React from 'react'
+import { withDesign } from 'storybook-addon-designs'
 import { Alert } from './Alert'
+import { Button } from '../Button/Button'
+import { alertTypes } from './Alert.levers'
+import { textMixed } from '../../.storybook/stories/Paragraph.stories'
+import DOMPurify from 'dompurify'
 
 export default {
   title: 'Composite/Alert',
-  component: Alert
+  decorators: [withDesign],
+  component: Alert,
+  subcomponents: { Button },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/Kmd4utmJFPRMVeCFEEBQhLtx/Decanter-Design-System?node-id=8095%3A0'
+    },
+    docs: {
+      description: {
+        component: 'For displaying a notification that keeps people informed of a status, or for displaying a validation message that alerts someone of an important piece of information.'
+      }
+    }
+  },
+  argTypes: {
+    type: {
+      control: {
+        type: 'select',
+        options: alertTypes
+      }
+    }
+  }
 }
 
 // Set up an Alert Template.
-const AlertTemplate = (args) => <Alert {...args}>{args.content}</Alert>
-
-// Some default text.
-const alertText = (
-  <React.Fragment>
-    This is the details of the alert message. This is a&nbsp;
-    <a href='https://news.stanford.edu'>link within an alert.</a>
-  </React.Fragment>
-)
+const AlertTemplate = ({ children, ...rest }) => {
+  // We do this to mimic sending in CMS content or another React component.
+  const content = <div dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(children) }} />
+  return (
+    <Alert {...rest}>{content}</Alert>
+  )
+}
 
 // Default State
+// /////////////////////////////////////////////////////////////////////////////
 export const Default = AlertTemplate.bind({})
-Default.args = { content: alertText }
+Default.args = {
+  children: textMixed,
+  heading: 'Alert Lorem Ipsum'
+}
+Default.parameters = {
+  jest: ['Alert.test.js']
+}
+
+export const Info = AlertTemplate.bind({})
+Info.args = {
+  children: textMixed,
+  heading: 'Alert Lorem Ipsum',
+  type: 'info'
+}
+
+// Supports Markdown.
+Info.parameters = {
+  docs: {
+    description: {
+      story: 'Informational style.'
+    }
+  }
+}
```

Comment:
This is how you get text above the example/documentation on the `docs` pages per story.